blockstore/apps/bundles/store.py

```diff
@@ -22,7 +22,7 @@
 from django.core.files.base import ContentFile, File
 from django.dispatch import Signal
 import attr
-from pyblake2 import blake2b
+from pyblake2 import blake2b  # pylint:disable=no-name-in-module
```

Comment:
Nit: [`pyblake2` is no longer needed](https://github.com/dchest/pyblake2). We could replace `pyblake2` with `hashlib` here, and remove this dependency.